tests/test_edgeql_for.py

```diff
@@ -346,7 +342,7 @@ async def test_edgeql_for_in_computable_02b(self):
         await self.assert_query_result(
             r'''
                 SELECT User {
-                    select_deck := (
+                    select_deck := ((
```

Comment:
This hunk seems to be unrelated.

Reply:
Yeah, though it's related in the sense that it's a follow-up to your multiplicity PR involving FOR